shacl/odrl-shacl.ttl

```diff
@@ -0,0 +1,147 @@
+@prefix odrl:  <http://www.w3.org/ns/odrl/2/> .
```

Comment:
I am just wondering if we should be including/testing with Shacl in this project. Is there perhaps a simpler example that we could provide that doesn't need Shacl?

I think this project has three aspects:

1. (optional use) creating and populating a SQL database (H2) for use in a Transformation
2. A method for executing Transformation tests via ScalaTest
3. (optional use) the ability to make Sparql assertions about the results

The reason why I have labelled (1) and (3) as optional is that not all transformations will need a SQL Database, and not all transformations will produce RDF. It would still be nice to test those that don't need (1) and/or (3).

Our tests of this project and examples should cover 1, 2, and 3. I am not certain that we need Shacl in the mix to cover that though?